### ISSUE TITLE
Pretrinketing Nevermelting Ice Crystal

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5908]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Pretrinketing was fixed a long time ago already but this specific trinket can still be used to pretrink it before fights and then being able to equip your normal trinket and still keep the buff from it, I remember I heard from someone on this bugtracker that it was like that during retail wotlk but he didnt bring any proof to that and also I haven't found any proof anywhere about being able to pretrinket this in any way, I can't see a single comment on any old forum that I checked about this being a thing.
**Expected behaviour**: Tell us what should happen instead
Pretty sure pretrinketing shouldn't be a thing since I can't find a single person saying anything about that anywhere on the web
**Steps to reproduce the problem**:

1. equip nevermelting ice crystal
2. use it
3. equip your normal trinket and see how the buff from NIC is still present

**Include proofs for this behaviour**
![Screenshot_3](https://user-images.githubusercontent.com/36212275/87062815-3acffe00-c216-11ea-8bc2-4ab2ced98188.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
item ID 50259
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
